service/src/main/kotlin/fi/espoo/evaka/daycare/DaycareQueries.kt

```diff
@@ -235,7 +235,7 @@ fun Database.Read.getOphUnitOIDs(): Map<DaycareId, String> =
     createQuery {
             sql(
                 """
-SELECT id, oph_unit_oid AS oph_unit_oid
+SELECT id, COALESCE(oph_unit_oid, '') AS oph_unit_oid
 FROM daycare
 """
```

Comment:
Olisiko tämä parempi korjaus?
```suggestion
WHERE oph_unit_oid IS NOT NULL
"""
```